reference/tiflash/deploy.md

```diff
@@ -57,79 +57,88 @@ This section describes how to install and deploy TiFlash in the following scenar
 
 ### Fresh TiFlash deployment
 
-For fresh TiFlash deployment, it is recommended to deploy TiFlash by downloading an offline installation package. The steps are as follows:
+TiUP Cluster is the deployment tool for TiDB 4.0 or later versions. It is recommended that you use TiUP Cluster to install and deploy TiFlash. The steps are as follows:
```

Comment:
```suggestion
TiUP cluster is the deployment tool for TiDB 4.0 or later versions. It is recommended that you use TiUP cluster to install and deploy TiFlash. The steps are as follows:
```
TiUP deployment doc uses `TiUP cluster`. Please update this word in other places (if any). @ran-huang